packages/manager/src/features/CloudPulse/Overview/GlobalFilters.tsx

```diff
@@ -122,6 +124,11 @@ export const GlobalFilters = React.memo((props: GlobalFilterProperties) => {
           </Tooltip>
         </Grid>
       </StyledGrid>
+      <CloudPulseDashboardFilterBuilder // this simple integration is for test purpose, the functionality integration will be done in upcoming PR by driving all filters through filterbuilder
+        dashboard={selectedDashboard!}
+        emitFilterChange={() => {}}
+        isServiceAnalyticsIntegration={false}
+      ></CloudPulseDashboardFilterBuilder>
```

Comment:
Can we do this because there are no children?

```suggestion
      <CloudPulseDashboardFilterBuilder // this simple integration is for test purpose, the functionality integration will be done in upcoming PR by driving all filters through filterbuilder
        dashboard={selectedDashboard!}
        emitFilterChange={() => {}}
        isServiceAnalyticsIntegration={false}
      />
```

Reply:
done

Reply:
done

Reply:
done